xt_imager.py

```diff
@@ -90,7 +89,11 @@ def main():
 
 class PYTFTPServer(object):
     def __init__(self, folder):
-        self.tftp_server = tftpy.TftpServer(folder)
+        try:
+            import tftpy
+            self.tftp_server = tftpy.TftpServer(folder)
+        except ImportError:
+            log.error("You need to install tftpy: 'pip3 install tftpy'")
```

Comment:
Idea is good, but message is not the best one. It should look like this

```
"Python tftpy module is not found. Please install it using yours distro package manager (i.e. 'apt install tftpy')
or locally with pip: 'pip3 install --user tftpy`"
```

Reply:
I have updated the message, but this needs additional explanations; maybe you will find some errors in my understanding.

1. I was not able to find a package for `apt` to install (tftpy, python-tftpy, python3-tftpy). On the tftpy page, only pip is mentioned as the way to install.
2. If the user wants to use the script's internal TFTP server, then the script has to be run under root to access port 69. As a result, we need to install tftpy under sudo to use it.

Reply:
Thanks, this is the easiest option. I have rewritten text according to your comment.